### PR TITLE
Add code to undivided rest of amount distribution

### PR DESCRIPTION
**Pull request by harzo**
Format code

### CODE REVIEW

File: test/holder.test.ts [hunk 1]
```diff
@@ -194,7 +180,7 @@ contract('PragmaticHodlings', accounts => {
       await hodlings.settleToken(token.address, { from: owner });
 
       for (const [idx, hodler] of hodlers.entries()) {
-        assertNumberAlmostEqual(
+        assertNumberEqual(
           await token.balanceOf(accounts[idx]),
           hodler.expectedAmount,
           1
```
Comment (kruligh) [inline test/holder.test.ts]: Remove third parameter. In almost equality it is fault tolerant epsilon.

**Comment by harzo** [comment]
@kruligh done :+1:

**Comment by kruligh** [comment]
@harzo I made tests correction.

**Comment by harzo** [comment]
@kruligh Ok. We could merge this now.

**Pull request merged**